alfasim_sdk/variables.py

```diff
@@ -46,6 +46,20 @@ class Scope(Enum):
 
 @attr.s(kw_only=True)
 class SecondaryVariable:
+    """
+    SOME TEXT
```

Comment:
Secondary variables are those variables that are not unknowns from the nonlinear system (That is, they are not directly solved in the nonlinear system), but they are calculated based on the nonlinear system results.